### DISCUSSION TITLE
Coffea Update Schedule

### DISCUSSION THREAD

**Discussion by aperloff** [CoffeaTeam/coffea-casa#135] [UNL tech support]
Is there an coffea update schedule for coffea-casa? When I login right now all I see if the “Coffea Base Image” for HATS2021 with Coffea 0.7.2. The latest version of Coffea is 0.7.4, which has an update to the TreeMakerSchema that we use. Are updates rolled out regularly or a way for people to request a new image? Maybe here or on the Issues tab?

**Comment by oshadura** [comment]
@aperloff thanks a lot for asking! We are working on update: such update is usually very quick and done in automatic way, but we have an issue with compatibility of new version of xrootd 5.2.0 required by coffea 0.7.4 and our custom coffea-casa xrd plugin. ETA for update: mid next week. Thanks for patience!
